products/health-care/digital-health-modernization/mhv-to-va.gov/landing-page/assets/links.md

```diff
@@ -0,0 +1,67 @@
+| My medications and allergies                | https://mhv-syst.myhealth.va.gov/mhv-portal-web/my-complete-medications-list |       |
+| Download my medical record (Blue Button)    | https://mhv-syst.myhealth.va.gov/mhv-portal-web/download-my-data             |       |
+| Lab and test results                        | https://mhv-syst.myhealth.va.gov/mhv-portal-web/labs-tests                   |       |
+| Health history                              | https://mhv-syst.myhealth.va.gov/mhv-portal-web/health-history               |       |
+| File a claim for travel reimbursement       | https://dvagov-btsss.dynamics365portals.us/signin                            |       |
+| Pay your copay bill                         | https://pay.gov/public/form/start/25987221                                   |       |
+| Order hearing aid batteries and accessories | https://www.va.gov/health-care/order-hearing-aid-batteries-and-accessories   |       |
+| Order prosthetic socks                      | https://www.va.gov/health-care/order-prosthetic-socks/                       |       |
+| Current veteran copay rates                 | https://www.va.gov/manage-va-debt/summary/copay-balances/                    |       |
+| Mental health services                      | https://www.va.gov/health-care/health-needs-conditions/mental-health         |       |
+| Dental care                                 | https://www.va.gov/health-care/about-va-health-benefits/dental-care/         |       |
+| Community Care                              | https://www.va.gov/COMMUNITYCARE/programs/veterans/index.asp                 |       |
+| Update my health benefits info (10-10EZR)   | https://www.va.gov/health-care/update-health-information/                    |       |
+| My health information card                  | https://mhv-syst.myhealth.va.gov/mhv-portal-web/health-information-card      |       |
+| The PACT Act and your benefits              | https://www.va.gov/resources/the-pact-act-and-your-va-benefits/              |       |
+| Check your mental health                    | https://mhv-syst.myhealth.va.gov/mhv-portal-web/check-your-mental-health     |       |
+| Veterans Health Library                     | https://www.veteranshealthlibrary.va.gov/                                    |       |
+| Healthy Living Centers                      | https://www.myhealth.va.gov/healthy-living-centers                           |       |
+| The My HealtheVet community                 | https://www.myhealth.va.gov/mhv-community                                    |       |
+| VA's Whole Health living                    | https://www.va.gov/wholehealth/                                              |       |
+| How to use VA Video Connect                 | https://www.myhealth.va.gov/ss20200320-va-video-connect                      |       |
+| Five Exercises for Balance                  | https://www.va.gov/my-health/#FIXME-need-link                                |       |
```

Comment:
There are some `#FIXME-` links because the *In the Spotlight* section had no links in the Excel File